pkg/tfbridge/auto_aliasing.go

```diff
@@ -51,14 +53,8 @@ import (
 // detected, and then overrides are cleared. Effectively this makes sure that upstream
 // MaxItems changes are deferred until the next major version.
 //
-// Panics if [ProviderInfo.ApplyAutoAliases] would return an error.
-func (info *ProviderInfo) MustApplyAutoAliases() {
-	err := info.ApplyAutoAliases()
-	contract.AssertNoErrorf(err, "Failed to apply aliases")
-}
-
-// See [MustApplyAutoAliases].
-func (info *ProviderInfo) ApplyAutoAliases() error {
+//go:linkname ApplyAutoAliases
```

Comment:
Does the linkname directive here mean that it is still also a method on ProviderInfo?

Also what about MustApplyAutoAliases - isn't that now missing?

Reply:
That linkname directive declares an object in `/pkg/tfbridge`: `ApplyAutoAliases` to the compiler. It's a hint that we might want to link against this object later.

Reply:
We don't export `MustApplyAutoAliases`, but that isn't breaking since we never did. We still have the method `MustApplyAutoAliases`: https://github.com/pulumi/pulumi-terraform-bridge/blob/de2933ba29e365331516d1888cf4ee75a060c5a6/pkg/tfbridge/info/external_methods.go#L39-L42